projects/view/client/singleProject/singleProjectDetailed.html

```diff
@@ -0,0 +1,60 @@
+<template name="singleProjectDetailed">
```

Comment:
In this, and all, file(s), use two spaces per level of indentation. Convert tabs to spaces.